### ISSUE TITLE
Getting Test Loss Information

### ISSUE THREAD

**Issue by abdksyed** [open-mmlab/mmocr#616]
I am trying to train `SATRN_small`.
I generated some text data with various fonts using `TDRG`.
The loss value is going down pretty quickly which is good, but the Test Accuracy right from 1st epoch is around 37%. I am not able to know why, is there any where to see the `Test Loss`, because the evaluation is showing different `accuracy`. So it's difficult to know if the model has overfit or something.

Also, generally on training for very long duration, the training loss should go on decrease, as to overfit the data, but I see that even the training loss will reach somewhere around `0.001` and keep fluctuation there.

**Comment by gaotongxiao** [comment]
We don't compute test loss, but you can compute the loss in `simple_test()` like how `forward_train()` does, and print the loss dict. They can be found here:
https://github.com/open-mmlab/mmocr/blob/7571763376c5c1b21cdd95461440f7db7dcf295d/mmocr/models/textrecog/recognizer/encode_decode_recognizer.py

Referring to the past training log, I'd speculate that your model was overfitting since your training loss is already much lower than ours. If your dataset is not large enough, consider reducing the parameters to avoid overfitting. Also, the quickest way to check if model is overfitting is to append training set as a part of test set.
